units/how_to_convert_to_json.txt

```diff
@@ -0,0 +1,7 @@
+A brief guide on how to convert to JSON
+=======================================
+1. Open the CSV file in some kind of CSV editor (not a text editor).
+2. Search and replace all commas in the description with temporary, unused character. I used pipe. 
+3. Save the file.
+4. Run python3 parser.py
+5. (Optional) Open the units.json file in a text editor. Search and replace all pipes back to commas.
```

Comment:
we could probably avoid this by using `"` around all the CSV values and escaping the `"` within the strings; then we wouldn't have to do steps 2 and 5.

Reply:
To clarify, the issue is that the CSV file has commas in the textual description of the unit. As a result, the game itself parses the CSV by ignoring everything in the description. Strangely enough, LibreOffice Calc is smart enough to figure it out as well. 
Once we parse it and convert it to JSON, the JSON API in either C++ or python is smart enough to deal with anything (obviously).

Reply:
@royfalk yes, one gets around the commands in the field by quoting the fields per the CSV specification (same with TSV and tabs in the field).